app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,36 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  def reviewer_notification(comment, reviewers)
     @comment = comment
     @proposal = comment.proposal
 
     # Email all reviewers of this proposal if notifications is true unless they made the comment
-    bcc = @proposal.ratings.map do |rating|
-      user = rating.user
-      if rating.event_teammate.try(:should_be_notified?) && @comment.user_id != user.id
-        user.email
-      end
+    to = reviewers.map do |reviewer|
+      reviewer.email if @comment.user_id != reviewer.id
```

Comment:
we can remove the @comment.user_id to reviewer.id check since the verify method will only send speaker comments through this mailer.  You do still need the map call for email.  :)

Also, you can shorten it with the symbol.to_proc call: 

to = reviewers.map(&:email)